Generator/Glitched-World/Checks/Overworld/Snowpeak Province/Snowpeak Freezard Grotto Chest.jsonc

```diff
@@ -1,5 +1,5 @@
 {
-    "requirements": "true",
+    "requirements": "CanDefeatIceKeese and CanDefeatFreezard and Ball_and_Chain",
```

Comment:
You do not need to defeat the Keese to access this chest. This logic seems wrong.

Reply:
Technically it doesn't affect logic at all because you need Shadow Crystal to enter the room and crystal also satisfies CanDefeatIceKeese. May have just been a mis-input on whoever added the logic.

Reply:
Can we go ahead and remove `CanDefeatIceKeese` or make whatever change is necessary for it to read correctly?